src/odbc/odbc.c

```diff
@@ -1398,14 +1398,74 @@ SQLGetEnvAttr(SQLHENV henv, SQLINTEGER Attribute, SQLPOINTER Value, SQLINTEGER B
 			exit; \
 	} while(0)
 
+static SQLRETURN
```

Comment:
These functions handle adding new parameters to the table

Reply:
Should this be in `mem.c`?

Reply:
Do you really need to copy and allocate for all columns every time?